go/Makefile

```
@@ -0,0 +1,16 @@
+RELEASE_DIR = ./pkg/utils/langfiles/resources
+RELEASE_FILES = ../resources/languages.yml ../resources/languages-customization.yml
+
+ifdef OS
+   CP = copy /y
+else
+   CP = cp -f
+endif
+
+init: copy build
+
+copy: 
+	mkdir $(RELEASE_DIR); for u in $(RELEASE_FILES); do $(CP) $$u $(RELEASE_DIR); done
```

```suggestion
	mkdir -p $(RELEASE_DIR); for u in $(RELEASE_FILES); do $(CP) $$u $(RELEASE_DIR); done
```

to make sure that it can be executed even if `$(RELEASE_DIR) already exists.